### PR TITLE
Added Labels to Resolve Issue #156

### PR DESCRIPTION
**Pull request by AlyssaVandale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,3 +126,64 @@ https://www.activestate.com/products/tcl/downloads/
...
+- Examples: priority: now, priority: 2day, priority: soon
+
+State Labels:
+
+- These labels describe the decision state of the issue or pull request.
+- Examples: state: approved, state: blocked, state: inactive, state: pending
+      - state: approved - this means that it is approved to proceed
+      - state: blocked - this means that something is blocking this action from happening
+      - state: inactive - this means no action is needed or possible, the issue is either fixed or addressed better in other issues
+      - state: pending - this means that the action is being done but a few things need to happen first
```
Comment (krerkkiat) [inline README.md:158]: Please take a look at the rendering at https://github.com/AlyssaVandale/examples#labels-and-their-meanings. I believe one space between 154 and 155 should fix it. This occur at other section down below as well.

**Review by krerkkiat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AlyssaVandale) [reply README.md:158]: oh i didn't even realize that, i will fix that. other than that, is there anything else that needs fixed?

Comment (krerkkiat) [reply README.md:158]: Everything else looks good!

Comment (AlyssaVandale) [reply README.md]: i fixed the indent problem, next time i will make a new branch for the issue. Thank you!

**Review by krerkkiat — APPROVED** [review]
Looks good now.

**Pull request merged**